packages/domain/lib/src/di/use_case_module.dart

```diff
@@ -51,4 +56,5 @@ void initUseCases() {
   inject.registerFactory(() => LogButtonUseCase(inject.get()));
   inject.registerFactory(() => LogPageUseCase(inject.get()));
   inject.registerFactory(() => FetchReviewsUseCase(inject.get()));
+  inject.registerFactory(() => UpdateAppUseCase());
```

Comment:
u don't need inject.get(); here?

Reply:
u r right

Reply:
nope, here I registering use case as dependency in ServiceLocator